project-files/XML/DCC-signed.xml

```diff
@@ -564,4 +564,4 @@
EerXQ9s3S5EXCWji4QBZ8yfvou9tSRYZuYbE1k4hJL8kFF9eaOdQd4HF0sW0WnATKoqB2zb2+pX6LprhHW9jSkN7GDv2dBdbKghCk1Q==</ds:SignatureValue><ds:KeyInfo><ds:X509Data><ds:X509Certificate>MIIEGzCCAwOgAwIBAgIUOwSIQLAeHowl6JRM3gYkUTWhSUQwDQYJKoZIhvcNAQELBQAwgZwxCzAJBgNVBAYTAkZJMRAwDgYDVQQIDAdVdXNpbWFhMREwDwYDVQQHDAhIZWxzaW5raTEZMBcGA1UECgwQQWFsdG8gVW5pdmVyc2l0eTERMA8GA1UECwwIU21hcnRjb20xFDASBgNVBAMMC0phYW5UYXBvbmVuMSQwIgYJKoZIhvcNAQkBFhVqYWFuLnRhcG9uZW5AYWFsdG8uZmkwHhcNMjAwNDE3MTIxMTM5WhcNMjEwNDE3MTIxMTM5WjCBnDELMAkGA1UEBhMCRkkxEDAOBgNVBAgMB1V1c2ltYWExETAPBgNVBAcMCEhlbHNpbmtpMRkwFwYDVQQKDBBBYWx0byBVbml2ZXJzaXR5MREwDwYDVQQLDAhTbWFydGNvbTEUMBIGA1UEAwwLSmFhblRhcG9uZW4xJDAiBgkqhkiG9w0BCQEWFWphYW4udGFwb25lbkBhYWx0by5maTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAPkSq2iCbQ69ZFNeXWPRYO9ogU7Is20JxFFP5Vi0nHh1fnHAIRLyn0dFp53RIVOQSDmqS5l4b9fqv8ms1GU8w6msnhU61X8Xgtz0fSwofUIFtHlhnKk7rgRMWq6mMvzmDHryduVbplxGxL//ZsUj8n7KTd+BWk1ALA/OPIubTJKUsF7XSEHbXhrb8ghuq/WyfQhtI56iT7lK+HxYJ8sm2rQmbGSGQAm7TpSFP3ObrDD1jTg5A1WCX5u4lenQpCe1Eo8wiZo3hjyvihswsbab/bDZM4us2qV9gUnWLq3MxyTf8jAx6eS/cga7IFaQHtU67rLEJQubBl8qz69/5npRpnECAwEAAaNTMFEwHQYDVR0OBBYEFLXUI7er1GHxQ3ZvtSsWPdGx36AMMB8GA1UdIwQYMBaAFLXUI7er1GHxQ3ZvtSsWPdGx36AMMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBAB0Yiym1FAlG6kyjVz0CJoQR11OGe2QJ1kEmGh3+Wh8N9C3txm9wHCBVoJwd0A1xTdbB4eu6HhUxKQOtAERQi0kjUwawu5IMwER0oskHsbKZ+lyqti4vim/oXah6DMnVS3UV7OcB3zJEZ2fFRfTJJw7xwrLgR4+KIE5ias8tQNiEYvEioP0DCU2HnY7ohZ7y+Pu5A7qmKUbcRSiMEwpGJZVWXVbBvrUe5KvlmlwBSSR0nst1x78xGlORtMYgB7lc3LD1PEKMh5jVoX/5wBDKmTupk/csfDwdMaRCEUW2fO6eY2Ryiuhg6Pt3soXwx+rAU4N+kEz6ArkPnr+q55KSiC4=</ds:X509Certificate></ds:X509Data></ds:KeyInfo><ds:Object><xades:QualifyingProperties xmlns:xades="http://uri.etsi.org/01903/v1.3.2#" Target="#id-e17196bfef4de1755afb64bc0f392cac"><xades:SignedProperties Id="xades-id-e17196bfef4de1755afb64bc0f392cac"><xades:SignedSignatureProperties><xades:SigningTime>2020-04-17T12:17:54Z</xades:SigningTime><xades:SigningCertificateV2><xades:Cert><xades:CertDigest><ds:DigestMethod Algorithm="http://www.w3.org/2001/04/xmlenc#sha512"/><ds:DigestValue>ZqwuN7Y/WHmXA9GFEIKNPwm7sBGjoPRI61QO1+1tOhPGV5pd6vv1a0mjO9/F99+VUsUs7MXqap+t5B9KxGsxPw==</ds:DigestValue></xades:CertDigest><xades:IssuerSerialV2>MIG7MIGipIGfMIGcMQswCQYDVQQGEwJGSTEQMA4GA1UECAwHVXVzaW1hYTERMA8GA1UEBwwISGVsc2lua2kxGTAXBgNVBAoMEEFhbHRvIFVuaXZlcnNpdHkxETAPBgNVBAsMCFNtYXJ0Y29tMRQwEgYDVQQDDAtKYWFuVGFwb25lbjEkMCIGCSqGSIb3DQEJARYVamFhbi50YXBvbmVuQGFhbHRvLmZpAhQ7BIhAsB4ejCXolEzeBiRRNaFJRA==</xades:IssuerSerialV2></xades:Cert></xades:SigningCertificateV2><xades:SignaturePolicyIdentifier><xades:SignaturePolicyImplied/></xades:SignaturePolicyIdentifier></xades:SignedSignatureProperties><xades:SignedDataObjectProperties><xades:DataObjectFormat ObjectReference="#r-id-e17196bfef4de1755afb64bc0f392cac-1"><xades:MimeType>text/xml</xades:MimeType></xades:DataObjectFormat></xades:SignedDataObjectProperties></xades:SignedProperties></xades:QualifyingProperties></ds:Object></ds:Signature></dcc:digitalCalibrationCertificate>
```

Comment:
Please do not mix changes in a single commit.  Please rewrite this commit so that it only changes `validation.js`

Reply:
I rewrote most of my commits. The history is much cleaner now. The history should be cleaner now. [https://github.com/JaanTaponen/DCC-XML-CSS-UI-demo/commits/validation-demo](https://github.com/JaanTaponen/DCC-XML-CSS-UI-demo/commits/validation-demo)

Reply:
Made some more comments directly to your branch, since I didn't notice that you had updated the PR.